openlibrary/plugins/upstream/addbook.py

```diff
@@ -753,9 +753,9 @@ def read_subject(subjects):
             f = io.StringIO(subjects.replace('\r\n', ''))
             dedup = set()
             for s in next(csv.reader(f, dialect='excel', skipinitialspace=True)):
-                if s.lower() not in dedup:
+                if s.casefold() not in dedup:
                     yield s
-                    dedup.add(s.lower())
+                    dedup.add(s.casefold())
```

Comment:
Hmmm I guess we can use casefolding instead of lower; a little more opportunity for unexpected deduping like Tom mentioned, but seems rather low risk/unlikely.